apps/studio/src/features/editing-experience/constants.ts

```diff
@@ -41,12 +44,11 @@ export const TYPE_TO_ICON: Record<
   video: BiMoviePlay,
   logocloud: BiCloud,
   [DYNAMIC_DATA_BANNER_TYPE]: TbApi,
+  [COLLECTION_WIDGET_TYPE]: TbDatabase,
   // TODO: Add in these new block types
   // table: BiTable,
   // divider: DividerIcon,
   // iframe-gmap,
   // iframe-formsg
   // iframe-youtube
 }
-
-export const REFERENCE_LINK_REGEX = /\[resource:(\d+):(\d+)\]/
```

Comment:
moved it to component lib since we also move `getResourceIdFromReferenceLink` there